### PR TITLE
chore: add e2e playwright to ci

### PR DESCRIPTION
**Pull request by mswertz**

### CODE REVIEW

**Review by MaxPostema — CHANGES_REQUESTED** [review]
I feel like we are making a bit of a mess in the living room.

'e2eTests' and 'tests-examples' Is adding a  "camelCase"  and "kebab-case" directory.

to more fit into the style of the repo maybe:
'e2e/tests' and 'e2e/samples'

Or use "kebab-case" for both directories, we already use it for "helm-chart".

File: tests-examples/demo-todo-app.spec.ts [hunk 1]
```diff
@@ -0,0 +1,437 @@
+import { test, expect, type Page } from '@playwright/test';
```
Comment (chinook25) [inline tests-examples/demo-todo-app.spec.ts:1]: This file can be deleted I'd say.

**Review by chinook25 — APPROVED** [review]
Looks fine to me, not sure how I can test whether the CI works.

**Comment by mswertz** [comment]
> I feel like we are making a bit of a mess in the living room.
> 
> 'e2eTests' and 'tests-examples' Is adding a "camelCase" and "kebab-case" directory.
> 
> to more fit into the style of the repo maybe: 'e2e/tests' and 'e2e/samples'
> 
> Or use "kebab-case" for both directories, we already use it for "helm-chart".

LOL. I am happy to rename to e2e/tests and take it from there

File: e2e/README.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#
+
+We have e2e tests using playwright
+
+To install:
+```yarn install```
+
+To test against a localhost:8080 run:
+
+```npx playwright test```
```
Comment (chinook25) [inline e2e/README.md:10]: yarn?

File: e2e/settings.gradle
Comment (chinook25) [inline e2e/settings.gradle:1]: empty file?

**Pull request merged**